tests/interpolation/built_in_resolvers/test_dict.py

```diff
@@ -0,0 +1,393 @@
+import re
```

Comment:
wow, 400 lines of tests.
I guess breaking the interpolation tests was the right move :)

Reply:
> I guess breaking the interpolation tests was the right move :)

It was! :)